backend/db/utils.py

```diff
@@ -206,6 +206,41 @@ def refresh_derived_tables(
             break
 
 
+ENGINES = {
+    # 'n3c': get_engine(),
+    # 'public': get_engine(schema='public')
+}
+
+
+def get_engine(isolation_level='AUTOCOMMIT', schema: str = SCHEMA, local=False):
```

Comment:
## `get_engine()`
@Sigfried Gonna just leave the work we did in this open PR. This is a low priority issue now that we've resolved the phone call in the house and also got an action that closes extra idle connections.